src/plugins/dashboard/public/utils/get_dashboard_capabilities.ts

```diff
@@ -7,31 +7,22 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import type { KibanaPluginServiceFactory } from '@kbn/presentation-util-plugin/public';
-import type { DashboardStartDependencies } from '../../plugin';
-import type { DashboardCapabilitiesService } from './types';
+import { DashboardCapabilities } from '../../common';
+import { coreServices } from '../services/kibana_services';
 
-export type DashboardCapabilitiesServiceFactory = KibanaPluginServiceFactory<
-  DashboardCapabilitiesService,
-  DashboardStartDependencies
->;
-export const dashboardCapabilitiesServiceFactory: DashboardCapabilitiesServiceFactory = ({
-  coreStart,
-}) => {
+export const getDashboardCapabilities = (): DashboardCapabilities => {
   const {
     application: {
-      capabilities: { dashboard, maps, visualize },
+      capabilities: { dashboard },
     },
-  } = coreStart;
+  } = coreServices;
 
   return {
     show: Boolean(dashboard.show),
     saveQuery: Boolean(dashboard.saveQuery),
     createNew: Boolean(dashboard.createNew),
-    mapsCapabilities: { save: Boolean(maps?.save) },
     createShortUrl: Boolean(dashboard.createShortUrl),
     showWriteControls: Boolean(dashboard.showWriteControls),
-    visualizeCapabilities: { save: Boolean(visualize?.save) },
```

Comment:
`mapsCapabilities` and `visualizeCapabilities` weren't actually used anywhere - so, I removed them